cluster/gce/config-test.sh

```diff
@@ -440,6 +440,9 @@ SCHEDULING_ALGORITHM_PROVIDER="${SCHEDULING_ALGORITHM_PROVIDER:-}"
 # Optional: install a default StorageClass
 ENABLE_DEFAULT_STORAGE_CLASS="${ENABLE_DEFAULT_STORAGE_CLASS:-true}"
 
+# Optional: install an in-tree StorageClass with NTFS fsType for Windows
+ENABLE_WINDOWS_STORAGE_CLASS="${ENABLE_WINDOWS_STORAGE_CLASS:-true}"
```

Comment:
This should only be used for testing, so let's rename this to indicate that (and prevent naive users from accidentally enabling it).